dataplane/pkg/common/egress_interface.go

```diff
@@ -78,25 +78,36 @@ func parseProcFile(scanner *bufio.Scanner) (string, net.IP, error) {
 		if strings.TrimSpace(parts[1]) == "00000000" {
 			outgoingInterface := strings.TrimSpace(parts[0])
 			defaultGateway := strings.TrimSpace(parts[2])
-			ip := parseGatewayIP(defaultGateway)
+			ip, err := parseGatewayIP(defaultGateway)
+			if err != nil {
+				logrus.Printf("Error Processing gateway IP %v for outgoing interface: %v", defaultGateway, outgoingInterface)
+				break
+			}
 			logrus.Printf("Found default gateway %v outgoing: %v", ip.String(), outgoingInterface)
 			return outgoingInterface, ip, nil
 		}
 	}
 	return "", nil, errors.New("Failed to locate default route...")
 }
 
-func parseGatewayIP(defaultGateway string) net.IP {
+func parseGatewayIP(defaultGateway string) (net.IP, error) {
+	if len(defaultGateway) != 8 {
+		return nil, errors.New("Failed to parse IP from string")
```

Comment:
please, use lower case error message

Reply:
@denis-tingajkin I thought we had CI enforcing that style check?  Did I misremember?

Reply:
@denis-tingajkin Got it :)

Reply:
@denis-tingajkin no, this is because `errors` now from `github.com/pkg/errors`, I guess linter just can't track such kind of issues

Reply:
@lobkovilya oops! I forgot that we using `github.com/pkg/errors` instead of `errors` for now. Yes, you can be right.